templates/github/.github/workflows/scripts/install.sh.j2

```diff
@@ -191,9 +184,7 @@ fi
 
 pulp config create --base-url {{ pulp_scheme }}://pulp{% if pulp_scheme != 'https' %} --no-verify-ssl{% endif %} --api-root "$PULP_API_ROOT" --username "admin" --password "password"
 {% if test_cli -%}
-if [[ "$TEST" != "docs" ]]; then
-  cp ~/.config/pulp/cli.toml "${REPO_ROOT}/../{{ cli_package }}/tests/cli.toml"
-fi
+cp ~/.config/pulp/cli.toml "${REPO_ROOT}/../{{ cli_package }}/tests/cli.toml"
 {%- endif %}
```

Comment:
Can you move that whole block up to around line 30 to have the cli stuff in one place?

Reply:
Didn't seem to work. Don't try this again just now.